lib/build.js

```diff
@@ -971,7 +971,7 @@ const buildActs = (config, act_proofs) => {
       pass: true,
       rough: false,
       hasGas: true,
-      gas: gas_json.map(g => kast.format(g, true, true))
+      gas: gas_json.map(g => kast.format(g, /*isRaw=*/ true, /*mixFix=*/ true, /*colorize=*/ false, /*forSpec=*/ true))
```

Comment:
These inline comments are just documentation, right? Doesn't everybody use LSP by now, telling you the signature of the function? ;)

I'm just doubting that this approach scales.

Reply:
Haha ok, too late

Reply:
Heh, I'm just a boomer I guess :). Also what do you mean by LSP? [This](https://langserver.org/)?

Reply:
Yep that's the one.